docs/reference/commandline/cli.md

```diff
@@ -378,6 +378,37 @@ list of root Certificate Authorities.
 
 ## Examples
 
+### <a name="host"></a> Specify daemon host (-H, --host)
+
+You can use the `-H`, `--host` flag to specify one or more sockets to use when
+you invoke a `docker` command. You can use the following protocols:
+
+- `unix://`
+- `tcp://`
+- `ssh://`
```

Comment:
There's also `npipe://` on Windows. Perhaps we should use a table for this, something like;


```markdown
| Scheme                                 | Description               | Example                          |
|----------------------------------------|---------------------------|----------------------------------|
| `unix://[<path>]`                      | Unix socket (Linux only)  | `unix:///var/run/docker.sock`    |
| `tcp://[<IP or host>[:port]]`          | TCP connection            | `tcp://174.17.0.1:2376`          |
| `ssh://[username@]<IP or host>[:port]` | SSH connection            | `ssh://user@192.168.64.5`        |
| `npipe://[<name>]`                     | Named pipe (Windows only) | `npipe:////./pipe/docker_engine` |
```

| Scheme                                 | Description               | Example                          |
|----------------------------------------|---------------------------|----------------------------------|
| `unix://[<path>]`                      | Unix socket (Linux only)  | `unix:///var/run/docker.sock`    |
| `tcp://[<IP or host>[:port]]`          | TCP connection            | `tcp://174.17.0.1:2376`          |
| `ssh://[username@]<IP or host>[:port]` | SSH connection            | `ssh://user@192.168.64.5`        |
| `npipe://[<name>]`                     | Named pipe (Windows only) | `npipe:////./pipe/docker_engine` |


Perhaps we also need to mention what the defaults are, and that only the scheme is required when setting it as option;

- if nothing is specified, the default is `unix:///var/run/docker.sock` on Linux and macOS, and `npipe:////./pipe/docker_engine` on Windows
- `-H tcp://` uses default `tcp://localhost:2375` (or `tcp://localhost:2376` if TLS options are set)
- `-H unix://`  uses default `unix:///var/run/docker.sock`
- `-H npipe://` uses default  `npipe:////./pipe/docker_engine`

Reply:
Good idea, I added this